nshlib/nsh.h

```diff
@@ -1394,7 +1394,8 @@ int nsh_foreach_direntry(FAR struct nsh_vtbl_s *vtbl, FAR const char *cmd,
  *
  ****************************************************************************/
 
-#if defined(CONFIG_FS_PROCFS) && !defined(CONFIG_NSH_DISABLE_PIDOF)
+#if defined(CONFIG_FS_PROCFS) && \
+  (!defined(CONFIG_NSH_DISABLE_PIDOF) || !defined(CONFIG_NSH_DISABLE_PKILL))
```

Comment:
let's remove `(!defined(CONFIG_NSH_DISABLE_PIDOF) || !defined(CONFIG_NSH_DISABLE_PKILL))`? to avoid the new break in the future.

Reply:
Done